source/opt/fold.h

```diff
@@ -59,6 +59,29 @@ bool IsFoldableOpcode(SpvOp opcode);
 // Returns true if |cst| is supported by FoldScalars and FoldVectors.
 bool IsFoldableConstant(const analysis::Constant* cst);
 
+// Returns true if |FoldInstruction| will fold an instruction whose result type
```

Comment:
suggest changing "will" to "could".  It's more accurate, and it's in line with the method comment on IsFoldable.

Reply:
Done.